python/cugraph/cugraph/structure/symmetrize.py

```diff
@@ -84,6 +86,11 @@ def symmetrize_df(
     if multi:
         return result
     else:
+        warnings.warn(
+            "'multi' is deprecated as the removal of multi edges is "
+            "only supported when creating a multi-graph",
+            FutureWarning,
+        )
```

Comment:
I think I'm missing something: does this mean a user who wants a MultiGraph could also want to remove the multiple edges? Why would they want a MultiGraph without multiple edges?

Reply:
No, It means that the only way to drop multi edges is by creating a multi-graph. In fact prior to that, we were using the flag `multi` in the `symmetrize` function  to drop multi-edges (if creating a directed or undirected Graph) through the `_memory_efficient_drop_duplicates` function which is dask based. But we can't just get rid of that flag because some users might still have it in their code base if they were directly calling the `symmetrize` function directly

Reply:
Okay, thanks.  I think my concerns will go away when Vibhu's feedback is addressed.